app/src/androidTest/java/org/mozilla/tv/firefox/ui/robots/NavigationOverlayRobot.kt

```diff
@@ -171,11 +178,18 @@ class NavigationOverlayRobot {
         }
 
         fun linearNavigateToTelemtryTileAndOpen(interact: SettingsRobot.() -> Unit): SettingsRobot.Transition {
```

Comment:
Hmm, I don't remember why I wrote it this way, but now that both of these are changing and actually look almost the same, maybe we should replace this with `linearNavigateToSettings` + `openTelemetryTile` - what do you think?

Reply:
Yes I was thinking that, but they belong to a different robot, so you can't reuse `linearNavigateToSettings` there.  We should refactor somehow, but that's outside the scope of this PR I think.